### DISCUSSION TITLE
Filtering by wavelength

### DISCUSSION THREAD

**Discussion by RichardScottOZ** [eroots/gams#3] [Ideas]
This looks pretty cool...any chance of adding some basic wavelength type filtering in for the layperson?

**Comment by eroots** [comment]
The package isn't really meant to reproduce the functionality that your normal software packages should have, but it should be simple enough so I'll look into it.

**Comment by RichardScottOZ** [comment]
Cool, sounds good!  Be a nice illustrative starting point, too.  Thanks.

Even though work for a reasonable size company, budget isn't there for thousands a dollars a year subscriptions to everything for everyone, so I, personally, have never used one, in the 'expensive corporate geophysics package sense'.

**Reply by eroots** [reply]
I've added some basic filtering options in the dev branch for now since there's a minor dependency issue, but give it a try if you like.

**Comment by RichardScottOZ** [comment]
Thanks!  Will give it a go shortly.
